.rubocop.yml

```diff
@@ -58,3 +58,5 @@ Style/HashTransformKeys:
   Enabled: false
 Style/HashTransformValues:
   Enabled: false
+Layout/EndOfLine:
+   Enabled: False
```

Comment:
[SUGGESTION]: Kindly take into consideration not stopping the linter from doing its job. Instead, try abiding by its rules. As you can see in this image, there are 2 files which do not have an ending line - as required by the linter:
![Screenshot from 2023-04-10 23-13-39](https://user-images.githubusercontent.com/64814566/230989763-541b7f5b-ee3f-4a2e-ab71-85e0c99492ab.png)